### PR TITLE
"send" label changed to "next" in wallet

### LINKED ISSUE #9974: Change wallet 'send' button to 'next'
**Problem**
The 'Send' button in the wallet generated Tx flow leads to the Tx overview, where you can edit network fee and check details.

Let's change the button that leads to this screen from 'Send' to 'Next', so users know there is another step in the process. Otherwise they may worry they are signing the Tx without review.

Screenshot:
image

**Acceptance Criteria:**
- Use the 'next' label for the bottom sheet, rather than the send button.

### PR DESCRIPTION
**Pull request by vkjr**
fixes #9974 

### Summary
Label changed

#### Platforms

- Android
- iOS

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/wallet/send/views.cljs [hunk 1]
```diff
@@ -130,7 +130,7 @@
          [render-contact to from-chat?]]]
        [toolbar/toolbar
         {:center
-         {:label               :t/wallet-send
+         {:label               :t/next
```
Comment (flexsurfer) [inline src/status_im/ui/screens/wallet/send/views.cljs]: we can't just change label, because it can be "send" when you're sending request

**Review by flexsurfer — CHANGES_REQUESTED** [review]

File: src/status_im/ui/screens/wallet/send/views.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/wallet/send/views.cljs]: when we send tx in chat

Comment (vkjr) [reply src/status_im/ui/screens/wallet/send/views.cljs]: thanks!

Comment (vkjr) [reply src/status_im/ui/screens/wallet/send/views.cljs]: Updated logic

**Review by flexsurfer — APPROVED** [review]

**Comment by Serhy** [comment]
@vkjr All good when sending TX from Wallet and when sending TX in 1-1 chat to the user with ENS name set.

But there is one place where still 'Send' used instead of 'Next'. When sending TX from chat (after user shared address via 'Accept and share address' button).
To reproduce:
1) Recover account with funds (UserA) on Device1
2) Create new account on Device2 (UserB)
3) UserB adds UserA in contacts and both open 1-1 chat
4) UserB sends `Request transaction` request in 1-1 chat to UserA
5) UserA taps `Sign and send` button in received messages
**Actual result:**
Screenshot 2020-03-16 at 15 09 32

**Comment by vkjr** [comment]
@Serhy, thanks! Fixed that.

**Comment by churik** [comment]
Issue is fixed, tested triggering transactions from 1-1 chat, dapp and wallet.
LGTM.

**Comment by churik** [comment]
@vkjr you can merge

**Comment by vkjr** [comment]
done ;)

**Pull request merged**